protoc-gen-grpc-gateway/internal/gengateway/generator.go

```diff
@@ -22,11 +23,12 @@ type generator struct {
 	registerFuncSuffix string
 	allowPatchFeature  bool
 	standalone         bool
+	useOpaqueAPI       bool
 }
 
 // New returns a new generator which generates grpc gateway files.
 func New(reg *descriptor.Registry, useRequestContext bool, registerFuncSuffix string,
-	allowPatchFeature, standalone bool) gen.Generator {
```

Comment:
hi @kop I'm not a maintainer- only a consumer of this library. I'm looking forward to using this feature. However, I suggest something other than adding a "one-off" parameter on the `New` constructor. Perhaps a different constructor, or an enum param.

Reply:
I think this is fine. This is an internal package and can't be used by anyone else.